Gobabygo/Gobabygo.ino

```diff
@@ -37,71 +39,85 @@ float correctForDeadzone(float axis, float deadzone) {
 }
 
 
-// This funtion gradually accelerates a value toward a target
-float accelerateToValue(float currentVal, float targetVal, int timeElapsed, float acceleration) {
-  return currentVal + (targetVal - currentVal) * timeElapsed * acceleration;
+// This function will cap the car's acceleration
+float safeAcceleration(float targetVal, float currentVal, float maxAcceleration) {
+  if (abs(targetVal - currentVal) <= maxAcceleration){
+    return targetVal;
+  }
+  else if (targetVal - currentVal > maxAcceleration){
+    return currentVal + maxAcceleration;
+  }
+  else {
+    return currentVal - maxAcceleration;
+  }
```

Comment:
For safety reasons I think we will want to have different limits for acceleration and deceleration. For example allow a fast approach towards zero (from either direction) but limit the rate as its magnitude increases.